env/xorg.env

```diff
@@ -0,0 +1 @@
+DISPLAY=${XORG_DISPLAY}
```

Comment:
This should probably go into `common.env` since it's always needed, right?

Reply:
Well, in theory some "utility" type containers (like pulse) might want stuff from `common.env` but they don't necessarily need access to an X display.  I don't feel super strongly about it though; I'm happy to push this into `common.env` if we think it's better to have fewer files/less complexity.